Source/WebKit/UIProcess/XR/openxr/PlatformXROpenXR.cpp

```diff
@@ -188,6 +280,109 @@ void OpenXRCoordinator::initializeDevice()
     collectViewConfigurations();
 }
 
+void OpenXRCoordinator::initializeGraphicsBinding()
+{
+    if (!m_platformDisplay)
+        m_platformDisplay = WebCore::PlatformDisplaySurfaceless::create();
+    if (!m_glContext)
+        m_glContext = WebCore::GLContext::createOffscreen(*m_platformDisplay);
+
+    m_graphicsBinding = createOpenXRStruct<XrGraphicsBindingEGLMNDX, XR_TYPE_GRAPHICS_BINDING_EGL_MNDX>();
```

Comment:
Shouldn't this be done only if `m_extensions->isExtensionSupported(XR_MNDX_EGL_ENABLE_EXTENSION_NAME ""_span` returs true?

Reply:
Finally I added an early return